msmbuilder/preprocessing/normalize.py

```diff
@@ -0,0 +1,157 @@
+# Author: Evan N. Feinberg <efein19@gmail.com>
+# Contributors: Christian Schwantes <schwancr@gmail.com>m Robert McGibbon <rmcgibbo@gmail.com>, Kyle A. Beauchamp  <kyleabeauchamp@gmail.com>
+# Copyright (c) 2016, Stanford University
+# All rights reserved.
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+from __future__ import print_function, division, absolute_import
+import numpy as np
+import warnings
+from ..base import BaseEstimator
+from ..utils import check_iter_of_sequences, array2d
+from sklearn.base import TransformerMixin
+
+#-----------------------------------------------------------------------------
+# Code
+#-----------------------------------------------------------------------------
+
+class Normalize(BaseEstimator, TransformerMixin):
+    """Transform data to zero mean and unit variance
+
+
+    Attributes
+    ----------
+
+    means_ : array-like, shape (1, n_features)
+        Overall mean of each feature over all data.
+    stdevs_ : array-like, shape (1, n_features)
+        Overall standard deviation of each feature.
+    n_observations_ : int
+        Total number of data points fit by the model. Note that the model
+        is "reset" by calling `fit()` with new sequences, whereas
+        `partial_fit()` updates the fit with new data, and is suitable for
+        online learning.
+    n_sequences_ : int
+        Total number of sequences fit by the model. Note that the model
+        is "reset" by calling `fit()` with new sequences, whereas
+        `partial_fit()` updates the fit with new data, and is suitable for
+         online learning.
+
+    Note: Numerically stable method of computing variance adapted from:
+    https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
+    """
+
+    def __init__(self):
+
+        """
```

Comment:
`__ini__` generally doesn't have a docstring. Put this in the class docstring or as line comments `# (these)`